pandas-stubs/core/frame.pyi

```diff
@@ -1902,14 +1902,24 @@ class DataFrame(NDFrame, OpsMixin):
         fill_value: float | None = ...,
     ) -> DataFrame: ...
     @overload
+    def sum(
+        self,
+        axis: Literal[1, "columns"],
```

Comment:
Isn't the 3rd overload the same as this one but has a broader `axis` annotation? Is it possible to simply move the third overload up?

Reply:
Yes, that's right.  Changed in next commit